Gems/EditorPythonBindings/Code/Source/ActionManager/MenuManagerBus.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Outcome/Outcome.h>
+#include <AzCore/std/string/string.h>
+
+#include <AzToolsFramework/ActionManager/Menu/MenuManagerInterface.h>
+
+namespace EditorPythonBindings
+{
+    //! MenuManagerRequestBus
+    //! Bus to register menus and add menu items to them in the Editor via Python.
+    //! If writing C++ code, use the MenuManagerInterface instead.
+    class MenuManagerRequests : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
+
+        //! Register a new Menu to the Menu Manager.
+        virtual AzToolsFramework::MenuManagerOperationResult RegisterMenu(const AZStd::string& identifier,
+            const AzToolsFramework::MenuProperties& properties) = 0;
```

Comment:
uber nit: Is it possible to move the first param to the next line to be consistent with the other function declarations?